src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/bootstrap-create-modal/bootstrap-create-modal.component.html

```diff
@@ -69,9 +69,16 @@
                     i18n-placeholder
                     id="token"
                     formControlName="token"
-                    readonly>
-          </textarea>
+                    readonly
+                    #token></textarea>
         </div>
+        <button class="btn btn-primary mb-3 float-right"
+                i18n-aria-label="Copy to Clipboard"
+                title="Copy to Clipboard"
+                i18n-title
+                (click)="copyToClipBoard(token)">
+          <ng-container i18n>Copy to Clipboard</ng-container>
+        </button>
```

Comment:
Just curious: why not using (and fixing/improving, as it still uses `execCommand`) the existing `shared/directives/copy2clipboard-button.directive.ts`?

Reply:
done